gammapy/spectrum/models.py

```diff
@@ -43,10 +148,16 @@ class PowerLaw(SpectralModel):
     reference : float, `~astropy.units.Quantity` 
         :math:`E_0`
     """
-    def __init__(self, index, amplitude, reference):
+    DEFAULT_INDEX = 2.3 * u.Unit('')
+    DEFAULT_AMPLITUDE = 2.5 * 1e-12 * u.Unit('cm-2 s-1 TeV-1')
```

Comment:
It super anyoing to always have to define parameters (with units) when one just wants to try some methods (especially when prototyping). Like this you can just write

```
from gammapy.spectrum.models import PowerLaw
p = PowerLaw()
```

So if you don't have a strong argument against default parameters I would prefer to leave them in. It does not keep anyone from being explicit, does it? (And we can be explicit in the docs)